src/Chapter04.Tests/Listing04.26.IdenticalInFunctionality.Tests.cs

```diff
@@ -8,8 +8,8 @@ public class ProgramTests
         [TestMethod]
         public void MainTest()
         {
-            const string expected = 
-@"The area of the circle is: 0.00";
+            string expected = 
+$@"The area of the circle is: {0.00M}";
```

Comment:
Does this need to be a decimal rather than a double?
```suggestion
$@"The area of the circle is: {0.00}";
```